Sources/xcparse/Console.swift

```diff
@@ -38,6 +54,10 @@ class Console {
     // MARK: Shell
     // user3064009's answer on https://stackoverflow.com/questions/26971240/how-do-i-run-an-terminal-command-in-a-swift-script-e-g-xcodebuild
     @discardableResult func shellCommand(_ command: String) -> String {
+        if self.verbose == true {
```

Comment:
Why not use log.verbose message

Reply:
Didn't refactor this after I wrote the method.  Fixed.